templates/default/tomcat/share.xml.erb

```diff
@@ -3,7 +3,10 @@
 
     <Manager className="de.javakaffee.web.msm.MemcachedBackupSessionManager"
       memcachedNodes="<%= node['tomcat']['memcached_nodes'] %>"
-        requestUriIgnorePattern=".*\.(ico|png|gif|jpg|css|js)$"
-        transcoderFactoryClass="de.javakaffee.web.msm.serializer.kryo.KryoTranscoderFactory"
-        />
+      requestUriIgnorePattern=".*\.(ico|png|gif|jpg|css|js)$"
+      transcoderFactoryClass="de.javakaffee.web.msm.serializer.kryo.KryoTranscoderFactory"
```

Comment:
The template isnt installed by default, only if you specify memcache nodes. Its always been that way. Im just making some final changes to that template.